pkg/services/rpcsrv/server.go

```diff
@@ -917,7 +917,15 @@ func (s *Server) calculateNetworkFee(reqParams params.Params) (any, *neorpc.Erro
 		return 0, neorpc.WrapErrorWithData(neorpc.ErrInvalidParams, fmt.Sprintf("failed to compute tx size: %s", err))
 	}
 	size := len(hashablePart) + io.GetVarSize(len(tx.Signers))
-	var netFee int64
+	var (
+		netFee int64
+		// Verification GAS cost can't exceed this policy.
+		gasLimit = s.chain.GetMaxVerificationGAS()
+	)
+	if gasLimit > int64(s.config.MaxGasInvoke) {
```

Comment:
Rename `gasLimit`? The `gasLimit` is only for the execution cost, it would be exceeded by the size of the transaction (for example).

Reply:
Naming is hard! At least it should do the trick even with this name.